pages/jdl.md

```diff
@@ -300,11 +312,13 @@ entity A {
 ```
 If you're using the generator prior to v4.9.X, you'd need to use patterns like this `pattern('[A-Z]+'`).
```

Comment:
Maybe we can remove this now, JHipster 4.9 is quite old now. What do you think?

Reply:
Good point, I'll update the PR